crates/ef-testing/src/evm_sequencer/evm_state/v1.rs

```diff
@@ -26,68 +26,86 @@ use starknet_api::{
 use starknet_crypto::{poseidon_hash_many, FieldElement};
 
 use super::Evm;
-use crate::evm_sequencer::{
-    account::{AccountType, KakarotAccount},
-    constants::{
-        kkrt_constants_v1::{CONTRACT_ACCOUNT_CLASS_HASH, EOA_CLASS_HASH},
-        CHAIN_ID, ETH_FEE_TOKEN_ADDRESS, KAKAROT_ADDRESS,
+use crate::{
+    evm_sequencer::{
+        account::{AccountType, KakarotAccount},
+        constants::{ETH_FEE_TOKEN_ADDRESS, KAKAROT_ADDRESS},
+        sequencer::KakarotSequencer,
+        utils::{split_u256, to_broadcasted_starknet_transaction},
     },
-    sequencer::KakarotSequencer,
-    utils::{compute_starknet_address, split_u256, to_broadcasted_starknet_transaction},
+    starknet_storage,
 };
 
 impl Evm for KakarotSequencer {
     /// Sets up the evm state (coinbase, block number, etc.)
     fn setup_state(&mut self) -> StateResult<()> {
-        let coinbase = KakarotAccount::new(&self.address, &Bytes::default(), U256::ZERO, &[])?;
+        let coinbase_address = *self.address();
+        let coinbase = KakarotAccount::new(&coinbase_address, &Bytes::default(), U256::ZERO, &[])?;
```

Comment:
since this is a valid acct, its nonce should be 1?

Reply:
EOA's start at nonce 0 no?

Reply:
ah maybe yes